docs/topics/request-response.rst

```diff
@@ -200,12 +214,37 @@ Example::
         self.logger.info("Visited %s", response.url)
 
 In some cases you may be interested in passing arguments to those callback
-functions so you can receive the arguments later, in the second callback. You
-can use the :attr:`Request.meta` attribute for that.
+functions so you can receive the arguments later, in the second callback.
+The following example shows how to achieve this by using the
+:attr:`Request.cb_kwargs` attribute:
+
+::
+
+    # pass information to the next callback using the Request.cb_kwargs attribute
```

Comment:
This comment seems a bit unnecessary to me.

Reply:
:ok_hand: